libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp

```diff
@@ -466,17 +490,13 @@ bool AP_ExternalAHRS_InertialLabs::check_uart()
             last_gps_ms = now_ms;
         }
     }
-#if AP_BARO_EXTERNALAHRS_ENABLED
     if (GOT_MSG(BARO_DATA) &&
         GOT_MSG(TEMPERATURE)) {
         AP::baro().handle_external(baro_data);
     }
-#endif
```

Comment:
Removing these definitions doesn't look right.  I suspect this change was unintentional.